### ISSUE TITLE
Deployer takes two cycles to launch service

### ISSUE THREAD

**Issue by dgem** [Financial-Times/coco-fleet-deployer#26]
As per the logs, it seems that the deployer notices that the service needs to be re-created in cycle 1 but waits to cycle 2 to start it:
```
Apr 05 11:04:07 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:04:07 Starting deploy run
Apr 05 11:04:30 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:04:30 DEBUG Unit [public-concordances-api-sidekick@1.service] is  updated
Apr 05 11:04:30 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:04:30 INFO Service public-concordances-api-sidekick@1.service differs from the cluster version
Apr 05 11:04:30 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:04:30 INFO Destroying unit public-concordances-api-sidekick@1.service
Apr 05 11:04:30 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:04:30 INFO Creating or updating unit public-concordances-api-sidekick@1.service
Apr 05 11:04:30 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:04:30 DEBUG Unit [public-concordances-api-sidekick@2.service] is  updated
Apr 05 11:04:30 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:04:30 INFO Service public-concordances-api-sidekick@2.service differs from the cluster version
Apr 05 11:04:30 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:04:30 INFO Destroying unit public-concordances-api-sidekick@2.service
Apr 05 11:04:31 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:04:31 INFO Creating or updating unit public-concordances-api-sidekick@2.service
Apr 05 11:04:32 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:04:32 Finished deploy run
Apr 05 11:05:32 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:05:32 Starting deploy run
Apr 05 11:05:57 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:05:57 INFO Setting target state for public-concordances-api-sidekick@1.service to launched
Apr 05 11:05:57 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:05:57 INFO Setting target state for public-concordances-api-sidekick@2.service to launched
Apr 05 11:05:57 ip-172-24-137-7.eu-west-1.compute.internal bash[3146]: 2016/04/05 11:05:57 Finished deploy run
```

**Comment by tamas-molnar** [comment]
https://jira.ft.com/browse/UPP-1219

**Comment by tamas-molnar** [comment]
Fixed in 1.0.3
